service/src/test/java/bio/terra/workspace/service/workspace/flight/WorkspaceDeleteFlightTest.java

```diff
@@ -0,0 +1,105 @@
+package bio.terra.workspace.service.workspace.flight;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import bio.terra.stairway.FlightMap;
+import bio.terra.workspace.app.configuration.external.FeatureConfiguration;
+import bio.terra.workspace.common.utils.FlightBeanBag;
+import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
+import bio.terra.workspace.service.job.JobMapKeys;
+import bio.terra.workspace.service.workspace.flight.aws.DeleteAwsContextStep;
+import bio.terra.workspace.service.workspace.flight.aws.DeleteControlledAwsResourcesStep;
+import bio.terra.workspace.service.workspace.flight.azure.DeleteAzureContextStep;
+import bio.terra.workspace.service.workspace.flight.azure.DeleteControlledAzureResourcesStep;
+import bio.terra.workspace.service.workspace.flight.gcp.DeleteGcpProjectStep;
+import bio.terra.workspace.service.workspace.model.WorkspaceStage;
+import java.util.UUID;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.test.context.ActiveProfiles;
+
+// This doesn't extend BaseUnitTest because we don't need BaseUnitTestMocks
+@Tag("unit")
+@ActiveProfiles({"unit-test"})
+public class WorkspaceDeleteFlightTest {
+
+  @Test
+  void stepsForRawlsWorkspace() {
```

Comment:
I realize these aren't the most meaningful tests, but it seems useful to have spelled out something to the effect of 'this flight contains these steps in this order'.